appveyor.yml

```diff
@@ -0,0 +1,58 @@
+environment:
+  matrix:
+    # Windows & python 3.6
+    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+      PYTHON: "C:\\Python36-x64"
+      PYTHON_ARCH: "64"
+
+    # Windows & python 3.7
+    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+      PYTHON: "C:\\Python37-x64"
+      PYTHON_ARCH: "64"
+
+    # Ubuntu & python 3.6
+    - APPVEYOR_BUILD_WORKER_IMAGE: Ubuntu
+      PYTHON: "3.6"
+
+    # Ubuntu & python 3.7
+    - APPVEYOR_BUILD_WORKER_IMAGE: Ubuntu
+      PYTHON: "3.7"
+
+stack: python %PYTHON%
+
+install:
+  - "git submodule update --init --recursive"
+  - cmd: "SET PATH=%PYTHON%;%PATH%"
+  - cmd: "SET PATH=C:\\Python36-x64\\Scripts;%PATH%"
+  - cmd: "SET PATH=C:\\Python37-x64\\Scripts;%PATH%"
+  - "python -V"
+  - "pip --version"
+  - "pip install pep8"
+  - cmd: "python -m pip install mypy"
+  - cmd: "python -m  pip install -r requirements.txt"
+  - sh: "pip install mypy"
+  - sh: "pip install -r requirements.txt"
+  - sh: "sudo add-apt-repository -y ppa:deadsnakes/ppa "
+  - sh: "sudo apt-get update"
+  - sh: "sudo apt-get -y install python3.6-dev"
+  - sh: "sudo apt-get -y install python3.7-dev"
+
+
+build: off
+
+test_script:
+  - cmd: "test_windows.bat"
+  - sh: "export QT_QPA_PLATFORM=offscreen"
+  - sh: "bash test_linux.sh"
+
+after_test:
+  - cmd: "mypy zubax-kucher || ver > nul"
+  - sh: "(mypy zubax-kucher; exit 0)"
+  - cmd: "pep8 --max-line-length=120 --exclude libraries || ver > nul"
+  - sh: "(pep8 --max-line-length=120 --exclude libraries; exit 0)"
+  - cmd: "7z a zubax-kucher.zip *"
+  - sh: "7z a zubax-kucher.zip * &>/dev/null &"
+  - "appveyor PushArtifact zubax-kucher.zip"
+  - "pyinstaller pyinstaller.spec"
```

Comment:
Do I get it right that the original spec file did not require any modifications to produce a Windows executable?

Reply:
Yep, it also works with windows exe